dkc/core/models/folder.py

```diff
@@ -62,16 +62,17 @@ class Meta:
     # # Prevent deletion of quotas while a folder references them
     # quota = models.ForeignKey(Quota, on_delete=models.PROTECT)
 
+    @property
```

Comment:
I'm not sure about what a good practice on properties vs traditional methods should be. My personal heuristic is:
1. Requires parameters -> method
2. Has any side effects -> method
3. Is very cheap (i.e. no database query) -> property
3. Is very expensive -> method
4. Otherwise -> ???

Reply:
Now that I think more, I agree that this particular case should be a property. It's very similar to the `RelatedManager` that tack `QuerySet`-ish properties onto models.